package.json

```diff
@@ -460,6 +460,7 @@
     "@kbn/dashboard-enhanced-plugin": "link:x-pack/platform/plugins/shared/dashboard_enhanced",
     "@kbn/dashboard-markdown": "link:src/platform/plugins/shared/dashboard_markdown",
     "@kbn/dashboard-plugin": "link:src/platform/plugins/shared/dashboard",
+    "@kbn/dashboards": "link:src/platform/packages/shared/dashboards",
```

Comment:
Ideally, the new package lives under the `dashboards` folder, something like `link:src/platform/packages/shared/dashboards/dashboard-selector` or something like that. Then we have room for other dashboard packages in the same folder.